std/http/server.ts

```diff
@@ -381,12 +381,31 @@ export class Server implements AsyncIterable<ServerRequest> {
     mux.add(this.acceptConnAndIterateHttpRequests(mux));
     return mux.iterate();
   }
+
+  // Returns the addres which this server is listening on.
+  get url(): string {
+    const parts = this.listener.addr().address.split(':')
+    const port = parts.pop();
+    let hostname = parts.join(':');
+    if (hostname === '0.0.0.0' || hostname === '127.0.0.1' || hostname === '[::]' || hostname === '[::1]')
+      hostname = 'localhost'
```

Comment:
localhost does not always map to 127.0.0.1 or ::1 ... this condition should be removed and the IP used

Reply:
Ok, I removed the condition.

> 2. s.url to return "http://localhost:8000/"

just though you wanted it to say localhost